ocs_ci/deployment/deployment.py

```diff
@@ -397,6 +410,16 @@ def deploy_ocs_via_operator(self):
         channel = config.DEPLOYMENT.get("ocs_csv_channel")
         csv_name = package_manifest.get_current_csv(channel=channel)
         csv = CSV(resource_name=csv_name, namespace=self.namespace)
+        if (
+            config.ENV_DATA["platform"] == constants.IBMCLOUD_PLATFORM
+            and not live_deployment
+        ):
+            csv.wait_for_phase("Installing", timeout=720)
+            logger.info("Sleeping for 30 seconds before applying SA")
+            time.sleep(30)
+            link_all_sa_and_secret(constants.OCS_SECRET, self.namespace)
+            logger.info("Deleting all pods in openshift-storage namespace")
+            exec_cmd(f"oc delete pod --all -n {self.namespace}")
```

Comment:
Couldn't we use a `delete_all_pods` function here?

Reply:
I don't need to do anything else with this and it's not worth to even create ocp object here in order to call it. So one line command is easier option here.